server_modules/log.js

```diff
@@ -0,0 +1,17 @@
+exports.middleware = function(req, res, next) {
+
+	req.log = function(level, message) {
+		console.log(level + ' - ' + Date.now().toString() + ': ' + message);

```

Comment:
Date is now logged as a timestamp. Use `new Date().toISOString()` or `Date().toLocaleString()` instead?